src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/BarbarianUtils.java

```diff
@@ -0,0 +1,45 @@
+package com.minecolonies.coremod.entity.ai.mobs.util;
+
+import com.minecolonies.api.util.CompatibilityUtils;
+import com.minecolonies.coremod.entity.ai.mobs.barbarians.AbstractEntityBarbarian;
+import net.minecraft.entity.Entity;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * A few utils used for barbarians.
+ */
+public class BarbarianUtils
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)